frontend/app/model/Eventbrite.scala

```diff
@@ -207,10 +207,17 @@ object Eventbrite {
     val limitedAvailabilityText = "Last tickets remaining"
     val isLimitedAvailability = internalTicketing.exists(event => event.ticketsNotSold <= Config.eventbriteLimitedAvailabilityCutoff && !event.isSoldOut)
     val ticketsNotSold = internalTicketing.map(_.ticketsNotSold)
-
     val isSoldOut = internalTicketing.exists(_.isSoldOut)
-    val isBookable = status == "live" && !isSoldOut
-    val isPastEvent = status != "live" && status != "draft"
+
+    val isBookable = {
+      val hasBookableTicketClasses = ticket_classes.exists(_.sales_end < DateTime.now)
```

Comment:
@rtyley Updated based on our conversation, reckon we still need to compare against `sales_start` too?